docs/source/appendices/design_decisions.rst

```diff
@@ -113,25 +113,23 @@ occurs in a low-complexity region, but rather describes the final and
 unambiguous state of the resultant sequence.
 
 
-.. _interbase-coordinates-design:
+.. _inter-residue-coordinates-design:
 
-Interbase Coordinates
-@@@@@@@@@@@@@@@@@@@@@
+Inter-residue Coordinates
```

Comment:
I don't recall discussing this change, but I like it because it is more applicable to proteins.